src/Language/Haskell/Exts/Location.hs

```diff
@@ -56,6 +56,12 @@ instance Location Decl where
   location (SpecInlineSig    loc _ _ _ _)       = Just loc
   location (InstSig          loc _ _ _ _)       = Just loc
   location (AnnPragma        loc _)             = Just loc
+  location (ClosedTypeFamDecl loc _ _ _ _)      = Just loc
+  location (PatSynSig         loc _ _ _ _ _)    = Just loc
+  location (PatSyn            loc _ _ _)        = Just loc
+  location (MinimalPragma     loc _)            = Just loc
+  location (RoleAnnotDecl     loc _ _)          = Just loc
```

Comment:
Oh great, they added constructors too, not just new arguments. So I guess the `{}` trick would *not* be sufficient to fix #149 after all.

Please bump up the `haskell-src-ext` dependency, as this will not compile with older versions which do not have those constructors.